doc/script_commands.txt

```diff
@@ -11021,6 +11025,19 @@ Examples:
 RandomIDArray, RandomValueArray, and RandomParamArray only works if the
 client (and server) supports the Item Random Options feature (PACKETVER >= 20150225).
 
+---------------------------------------
+
+*mesitemlink(<item_id>);
```

Comment:
Sometimes the displayed string is slightly different from the item name, for example
`<ITEM>["+.@new_item_name$+"]<INFO>" + .@new_item + "</INFO></ITEM>` in
https://github.com/rathena/rathena/pull/7565/files#diff-27beaaf26bd00dd9ba1e06e40bda9b99ffe6ae850d3f2fdb4f5dc9cfaefb8386R28
How about
```suggestion
*mesitemlink(<item_id>,{<string displayed>});
```